src/common/FilePreviewCommon/Formatters/FilePreviewJsonSerializerContext.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.PowerToys.FilePreviewCommon.Monaco.Formatters;
+
+[JsonSourceGenerationOptions(WriteIndented = true)]
+[JsonSerializable(typeof(JsonDocument))]
+internal sealed partial class FilePreviewJsonSerializerContext : JsonSerializerContext
```

Comment:
I remember I see this file twice on different location.
Looks like we need somewhere more centralized but not per project.

Reply:
@yeelam-gordon each class implementing the `JsonSerializerContext` class has to be in a spot where each of the class types are available to it. There are many types that are in the Settings project, but some types that are only in the individual projects. This one though, would actually be a good one to be moved to a more centralized location.

Reply:
Sure, move it to a centralized location is a good idea. But if we want to do it, we need to consider which location is appropriate and wether we need to move the relate model class (such as JsonDocument) to the same location. 

I think we need to have more discussion about this proposal (centrallized location to storage xxxSerializerContext class) but at least in this time, I think let it leave here is good enough.
@yeelam-gordon  @snickler